frontend/src/packages/articles/articles-api.package.ts

```diff
@@ -54,6 +55,40 @@ class ArticleApi extends HttpApi {
 
     return await response.json<ArticleBaseResponseDto>();
   }
+
+  public async share(id: string): Promise<{ link: string }> {
+    const response = await this.load(
+      this.getFullEndpoint(ArticlesApiPath.SHARE, { id }),
+      {
+        method: 'POST',
+        contentType: ContentType.JSON,
+        payload: JSON.stringify({ id }),
+        hasAuth: true,
+      },
+    );
+
+    const { link } = await response.json<{ link: string }>();
+
+    await writeTextInClipboard(link);
+
+    return { link };
+  }
+
+  public async getByToken(payload: {
+    token: string;
+  }): Promise<ArticleBaseResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(ArticlesApiPath.TOKEN, {}),
+      {
+        method: 'POST',
+        contentType: ContentType.JSON,
+        payload: JSON.stringify(payload),
```

Comment:
Pass it through params
```suggestion
      this.getFullEndpoint(ArticlesApiPath.TOKEN, { token }),
      {
        method: 'GET',
        contentType: ContentType.JSON,
```